src/main/resources/feedbackQuestionConstSumEditFormTemplate.html

```diff
@@ -11,34 +11,79 @@
                 </a>
             </div>
         </div>
-        
+
         <input type="hidden" name="${Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFCHOICECREATED}"
-            id="${Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFCHOICECREATED}-${questionNumber}" 
+            id="${Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFCHOICECREATED}-${questionNumber}"
             value="${numOfConstSumOptions}">
         <input type="hidden" name="${Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMTORECIPIENTS}"
-            id="${Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMTORECIPIENTS}-${questionNumber}" 
+            id="${Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMTORECIPIENTS}-${questionNumber}"
             value="${constSumToRecipientsValue}">
     </div>
     <div class="col-sm-6">
         <div class="form-inline">
             <div class="row">
-                <div class="col-md-5 col-lg-4">
-                    <label class="bold-label width-100-pc margin-top-7px margin-bottom-7px tablet-no-mobile-margin-top-0">
-                        <b>Points to distribute </b>
+                <div class="col-md-12">
+                    <label
+                        class="bold-label width-100-pc margin-top-7px margin-bottom-7px tablet-no-mobile-margin-top-0">
+                        <b>Total Points to distribute: </b>
                     </label>
                 </div>
-                <div class="col-md-4">
-                    <select class="width-100-pc select form-control margin-bottom-7px"
-                            disabled
-                            id="constSumPointsPerOptionSelect-${questionNumber}"
-                            name="${Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTSPEROPTION}">
-                        <option value="false">in total:</option>
-                        <option value="true" id="constSumOption_Option-${questionNumber}" ${selectedConstSumPointsPerOption}>per option:</option>
-                        <option value="true" id="constSumOption_Recipient-${questionNumber}" ${selectedConstSumPointsPerOption}>per recipient:</option>
-                    </select>
+
+                <div class="col-xs-12 margin-bottom-7px padding-left-35px">
+                    <div class="col-xs-1">
+                        <input type="radio" id="constSumPointsTotal-${questionNumber}"
+                            name="${Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTSPEROPTION}"
+                            value="false" checked>
```

Comment:
Missed this out earlier, but you should indent the `name="...` 8 spaces away from the `<input ...`, to better distinguish the attributes of a tag (8 spaces) as compared to its children/text content (4 spaces) eg.

```
<input type="radio" id="constSumPointsTotal-${questionNumber}"
        name="${Const.ParamsNames.FEEDBACK_QUESTION_CONSTSUMPOINTSPEROPTION}"
        value="false" checked>
```

This applies to several other elements in this file as well.